cpp/pipeline/PhaseAddAuthorSynonyms.cc

```diff
@@ -0,0 +1,136 @@
+#include "PhaseAddAuthorSynonyms.h"
+#include <iostream>
+#include <vector>
+#include <cstdlib>
+#include "Compiler.h"
+#include "MarcUtil.h"
+#include "MediaTypeUtil.h"
+#include "StringUtil.h"
+#include "Subfields.h"
+#include "util.h"
+
+
+static unsigned modified_count(0);
+static std::map <std::string, std::string> author_to_synonyms_map;
+static std::string field_list("100abcd:400abcd");
+static std::string primary_author_field("100abcd");
```

Comment:
Diese Variable sollte in PhaseAddAuthorSynonyms::process definiert sein und konstant und großgeschrieben sein.